tests/integration/compose/docker-compose-internal-auth_test.go

```diff
@@ -150,7 +159,7 @@ func (s *commonServicesInternalAuthDockerComposeUpTest) TestDockerComposeUp() {
 		{
 			"composeDedicatedPlugins",
 			internalAuthComposeDedicatedPluginsFile,
-			s.overrideFiles,
+			s.overrideFilesPlugins,
```

Comment:
For dedicated plugins, use the new dedicated plugin override files for the test case.